samples/snippets/subscriber.py

```diff
@@ -68,6 +68,95 @@ def list_subscriptions_in_project(project_id: str) -> None:
     # [END pubsub_list_subscriptions]
 
 
+def pubsub_subscribe_otel_tracing(
+    subscription_project_id: str,
+    cloud_trace_project_id: str,
+    subscription_id: str,
+    timeout: Optional[float] = None,
+) -> None:
+    """
+    Subscribe to `subscription_id` in `subscription_project_id` with OpenTelemetry enabled.
+    Export the OpenTelemetry traces to Google Cloud Trace in project
+    `trace_project_id`
+    Args:
+        subscription_project_id: project ID of the subscription.
+        cloud_trace_project_id: project ID to export Cloud Trace to.
+        subscription_id: subscription ID to subscribe from.
+        timeout: time until which to subscribe to.
+    Returns:
+        None
+    """
+    # [START pubsub_subscribe_otel_tracing]
+    from opentelemetry import trace
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import (
+        BatchSpanProcessor,
+    )
+    from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+    from opentelemetry.sdk.trace.sampling import TraceIdRatioBased, ParentBased
+
+    from google.cloud import pubsub_v1
+    from google.cloud.pubsub_v1 import SubscriberClient
+    from google.cloud.pubsub_v1.types import SubscriberOptions
+
+    # TODO(developer)
+    # subscription_project_id = "your-subscription-project-id"
+    # subscription_id = "your-subscription-id"
+    # cloud_trace_project_id = "your-cloud-trace-project-id"
+    # timeout = 300.0
+
+    # In this sample, we use a Google Cloud Trace to export the OpenTelemetry
+    # traces: https://cloud.google.com/trace/docs/setup/python-ot
+    # Choose and configure the exporter for your set up accordingly.
+
+    # Sample 1 in every 100 traces
+    sampler = ParentBased(root=TraceIdRatioBased(1 / 100))
```

Comment:
Same comment as publisher, make this default to 100% sampling.

Reply:
Done.